### ISSUE TITLE
Create a queue of animation

### ISSUE THREAD

**Issue by chinhodado** [wout/svg.js#225]
Is there a way to create a queue of animation instead of using `after()`?

`after()` makes the code complex with a large number of animations that has to be done in sequence. 

Plus, I'm not sure how to use `after()` with dynamically generated animations. For example, if I have a loop like this

    for (var i = 0; i < 10; i++) {
        element.animate({duration: i}).move(i, i);
    }

how can I use `after()` to have the animations done in sequence?

**Comment by lweigand** [comment]
Is this implemented already?
I think it would be cool if you could simply chain `.animate()` calls like this:
```javascript
element
  .animate(300).move(10, 10)
  .animate({ duration: 500 }).fill('#f06')
  .animate({ ease: '>' }).size(50, 50)
```
I believe the current behaviour of this is to stop an already running animation, though this is not really needed, as one can always use `.stop()`
This would also make sense as it uses the same code that is already being used for animation.

**Comment by Fuzzyma** [comment]
This was stalled long time but hey - take a look at the 3.0.0 branch when you want to go the first steps with the new fx module. Until now only the move function works and its surely not ready. But in case you are curious just try it out!

**Comment by Fuzzyma** [comment]
implemented in 2.3.0
